JS/company.js

```diff
@@ -0,0 +1,74 @@
+
+    window.onload = (e) => {
+        e.preventDefault();
+        let urlParams = new URLSearchParams(window.location.search);
+        symbol = urlParams.get('symbol');
+        getCompanyProfile(`https://stock-exchange-dot-full-stack-course-services.ew.r.appspot.com/api/v3/company/profile/${symbol}`)
```

Comment:
instead of rewriting the API adress, you should use global variable to hold this baseURL (https://stock-exchange-dot-full-stack-course-services.ew.r.appspot.com/api/v3/)

Reply:
Implemented in Milestone-6.